src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const REVERSE_ROBOT = {};
```

Comment:
It's better to use a more descriptive name for the new object, like 'invertedRobot' instead of 'REVERSE_ROBOT'.